cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,13 @@
 /// <reference types="cypress" />
 
+
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-    
+    cy.visit("/").get('h1').should('include.text' , 'conduit');
```

Comment:
separate these actions
```suggestion
    cy.visit('/');
    cy.get('h1').should('include.text' , 'conduit');
```